### PR TITLE
Changed example to restrict only resizing images from origin or host

### PR DESCRIPTION
**Pull request by evanderkoogh**
As mentioned in https://chat.google.com/room/AAAAGX7FDAU/U0WF41wf64I, one of our customers mentioned that the code is an XSS vulnerability when straight copy & pasted. I don't agree that it is a full XSS exploit, but it would certainly make it easier to get things wrong with a straight copy and paste of the code.

The example will now only take an absolute pathname and will make the relative to either the origin if one is present and the hostname if not. This should make the example more secure by default.

I haven't been able to test this yet, so it would be great if someone could test it. I just wanted to get a PR in so we wouldn't forget.

### CODE REVIEW

**Review by ganzcf — APPROVED** [review]

File: products/images/src/content/resizing-with-workers.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ if (response.ok) {
 
 ### An example worker
 
-Assuming you [set up a worker](https://developers.cloudflare.com/workers/learning/getting-started) on `https://example.com/image-resizing` to handle URLs like this: `https://example.com/image-resizing?width=80&image=https://example.com/uploads/avatar1.jpg`
+Assuming you [set up a worker](https://developers.cloudflare.com/workers/learning/getting-started) on `https://example.com/image-resizing` to handle URLs like this: `https://example.com/image-resizing?width=80&image=/uploads/avatar1.jpg`
```
Comment (deadlypants1973) [inline products/images/src/content/resizing-with-workers.md]:
```suggestion
Assuming you [set up a Worker](https://developers.cloudflare.com/workers/get-started/guide) on `https://example.com/image-resizing` to handle URLs like this: `https://example.com/image-resizing?width=80&image=/uploads/avatar1.jpg`
```

**Review by deadlypants1973 — COMMENTED** [review]
Updated URL for Workers get started guide.

**Review by deadlypants1973 — APPROVED** [review]

**Comment by lukeed** [comment]
There is a syntax error in these changes. Also @evanderkoogh I updated the example to address the XSS fix (#1911). In speaking to @ganzcf, adding a `hostname` check was enough to patch the vuln.

**Comment by deadlypants1973** [comment]
> There is a syntax error in these changes. Also @evanderkoogh I updated the example to address the XSS fix (#1911). In speaking to @ganzcf, adding a `hostname` check was enough to patch the vuln.

Can we close this then?

**Pull request closed without merging**